mne/viz/raw.py

```diff
@@ -132,13 +132,14 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
 
     remove_dc : bool
         If True remove DC component when plotting data.
-    order : 'type' | 'original' | 'selection' | 'position' | array of int
-        Order in which to plot data. 'type' groups by channel type,
-        'original' plots in the order of ch_names, 'selection' uses
-        Elekta's channel groupings (only works for Neuromag data) and
-        'positions' groups the channels by the positions of the sensors.
-        If array, the order is determined by the indices in the array.
-        Defaults to 'type'.
+    order : str | array of int
+        Order in which to plot data. 'type' groups by channel type, 'original'
+        plots in the order of ch_names, 'selection' uses Elekta's channel
+        groupings (only works for Neuromag data), 'position' groups the
+        channels by the positions of the sensors and 'lasso' uses the same
+        groups as 'position', but also allows lasso selection of custom
+        channels in the selection topomap. If array, only the channels in the
+        array are plotted in the given order. Defaults to 'type'.
```

Comment:
lasso works like a charm for me. I just feel that yet another parameter option is hard to discover.

is it possible to enable the lasso automatically for all methods? at least the ones that show you the channel locations.

one more nitpick I feel that line of the data are too thick. Can you try to make them thiner?